wiki/Submission/Beatmap_Ranking_Procedure/en.md

```diff
@@ -1,44 +1,23 @@
+In order for a beatmapset to receive a nomination, it must follow and be in accordance with the [ranking criteria](/wiki/Ranking_Criteria) and must have at least 5 hypes. Hypes can be obtained by having other users hype your beatmap in the Discussion page of the map, or under the map's description. Once a beatmap has at least 5 hypes, a BN can nominate it if they feel it meets two conditions:
 
-1.  The beatmap meets the [Ranking Criteria](/wiki/Ranking_Criteria). Basically, this must mean that the map has the correct [timing](/wiki/Beatmap_Editor/Timing), has at least 2 [difficulties](/wiki/Difficulties) (one of which is an [Easy](/wiki/Difficulties/osu!/Easy) or a [Normal](/wiki/Difficulties/osu!/Normal)), and is overall both mapped to the music and playable.
-2.  The beatmap meets overall quality standards that are not part of the Ranking Criteria. For example, if a slider is not very pretty, then a BN would likely request for it to be made prettier before bubbling it.
+1.  The beatmapset meets the [Ranking Criteria](/wiki/Ranking_Criteria).
+2.  The beatmapset meets overall quality standards that are not part of the Ranking Criteria.
 
-### Mode-Specific Bubbles
+## Nomination Reset
 
-Mode-Specific Bubbles are bubbles applied to hybrid mapsets for specific modes. The criteria for any of these bubbles is regulated to its respective [Ranking Criteria](/wiki/Ranking_Criteria), however only Mode-Specific BNs can apply these bubbles. For a list of available BNs, see [the respective list on its article](/wiki/People/Beatmap_Nomination_Group).
+Updating a nominated beatmapset will cause the nomination count to be reset. This means it will need to be nominated again. Alternatively, a nomination can be reset by another BN or QAT, if either the beatmapset does not follow one or more rules in the Ranking Criteria or the BN has issues with the quality of the map. If a beatmapset is reset due to quality standards not part of the Ranking Criteria, then it is called a veto. Vetoes are dealt with mediation by the QAT. 
 
-Popped Bubble ![Bubble pop icon](/wiki/shared/icon/bubble-pop.gif "Bubble pop icon")
----------------------------------------------------------------------------------
+Note: In the case of a minor change to a nominated beatmapset, then the BN can nominate the beatmapset again immediately, but in the case of a veto, a decision from the QAT needs to be done before the map is allowed to be nominated again.
 
-Updating a Bubbled beatmap will cause this icon to replace it, it means it needs to get into bubbled status again, alternatively a BN that disagrees with the judgment of the other staff member may place this icon as well. Some BNs place this icon on your map while modding it to prevent another BN to look at it, and is very likely it will get ranked afterwards.
+## Qualified and Ranked Sections
```

Comment:
```suggestion
## Qualified and Ranked Categories
```

Reply:
Will keep Sections, as that is the terminology used in the Ranking Criteria, and other mapping related wiki pages, like the BN rules https://celerih.s-ul.eu/5FwcjlJy.png